esmvalcore/_recipe.py

```diff
@@ -428,8 +446,10 @@ def _get_correct_fx_file(variable, fx_varname, config_user):
     # allow for empty lists corrected for by NE masks
     if fx_files:
         fx_files = fx_files[0]
```

Comment:
yes that's a good question - normally yes, one single file, but this will fail miserably when we actually find two or more files. Unfortunately can't do anything about this at this stage, we'll correct it in a different PR if we see otherwise